components/teleport/Teleport.vue

```diff
@@ -163,18 +163,19 @@ const { urlPrefix } = usePrefix()
 const fiatStore = useFiatStore()
 const fromChain = ref(Chain.POLKADOT) //Selected origin parachain
 const toChain = ref(Chain.ASSETHUBPOLKADOT) //Selected destination parachain
-const amount = ref() //Required amount to be transfered is stored here
+const amount = ref(0) //Required amount to be transfered is stored here
 const unsubscribeKusamaBalance = ref()
 const teleportFee = ref()
 
-const nativeAmount = computed(
-  () => (amount.value || 0) * Math.pow(10, currentTokenDecimals.value),
+const nativeAmount = computed(() =>
+  Math.floor(amount.value * Math.pow(10, currentTokenDecimals.value)),
 )
+
 const amountToTeleport = computed(() =>
-  nativeAmount.value === 0 ? 0 : nativeAmount.value - teleportFee.value,
+  Math.max(nativeAmount.value - teleportFee.value, 0),
 )
 
-const insufficientAmountAfterFees = computed(() => amountToTeleport.value < 0)
+const insufficientAmountAfterFees = computed(() => amountToTeleport.value <= 0)
```

Comment:
```suggestion
const insufficientAmountAfterFees = computed(() => amountToTeleport.value === 0)
```

it's never going to be less then 0